src/Components/App/App.js

```diff
@@ -19,6 +25,7 @@ function App() {
   }, [])
 
   function showSingleMovie(id) {
+    // setSingleMovie(null)
```

Comment:
Did you mean to comment this out? @judy0ye

Reply:
I meant to keep it in! Good catch!